### PR TITLE
added install swanky node command

### PR DESCRIPTION
**Pull request by shunsukew**
For those who initialize projects without local swanky node but later would like to use.

### CODE REVIEW

File: packages/cli/src/commands/node/install.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    await ensureSwankyProject();
+
+    const { flags } = await this.parse(InstallNode);
+    const config = await getSwankyConfig();
+    const spinner = new Spinner(flags.verbose);
+
+    const projectPath = path.resolve();
+
+    if (config.node.localPath !== "") {
+        const answers = await inquirer.prompt([
```
Comment (codespool) [inline packages/cli/src/commands/node/install.ts:25]: Let's try to gradually phase out inquirer.
This seems like a great use case for https://github.com/oclif/core/blob/main/src/cli-ux/README.md

**Review by codespool — CHANGES_REQUESTED** [review]

File: packages/cli/src/commands/node/install.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import inquirer = require("inquirer");
+import path = require("node:path");
+import { writeJSON } from "fs-extra";
+import { choice } from "../../lib/prompts";
+
+export class InstallNode extends Command {
+  static description = "Install swanky node binary";
+
+  static flags = {
+    verbose: Flags.boolean({ char: "v" }),
```
[the remark this answers is not in the document]
Comment (shunsukew) [reply packages/cli/src/commands/node/install.ts:12]: I forgot again. I'll update

**Comment by shunsukew** [comment]
Thank you so much for the feedback! 

> Additionally to this feature I would love to see some kind of swanky-node version locking mechanism (like package.lock.json) to share the same node across devs ends.

That would be a great feature, I assume that can be part of `swanky.config.json`. If version is already defined, install that version of node instead of the latest one.

File: packages/cli/src/commands/node/install.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+export class InstallNode extends BaseCommand<typeof InstallNode> {
+  static description = "Install swanky node binary";
+
+  async run(): Promise<void> {
+    await ensureSwankyProject();
+
+    const { flags } = await this.parse(InstallNode);
+    const config = await getSwankyConfig();
+    const spinner = new Spinner(flags.verbose);
```
Comment (codespool) [inline packages/cli/src/commands/node/install.ts:15]: Base command does this, you can access them through `this.swankyConfig` and `this.spinner`

**Review by codespool — CHANGES_REQUESTED** [review]

File: packages/cli/src/commands/node/install.ts (hunk 3 above)
Comment (codespool) [reply packages/cli/src/commands/node/install.ts:15]: Sorry for delayed review 😅

Comment (shunsukew) [reply packages/cli/src/commands/node/install.ts:15]: No problem.

> Base command does this, you can access them through this.swankyConfig and this.spinner

That's true. thank you for the review

**Review by codespool — APPROVED** [review]
LGTM

**Pull request merged**